AUTHORS.rst

```diff
@@ -10,4 +10,12 @@ Development Lead
 Contributors
 ------------
 
-None yet. Why not be the first?
+* Jakob Toftegaard
+* Niels Bassler
+* Toke Printz
+* Leszek Grzanka
+* Lukasz Jelen
+* Arkadiusz Cwikla
+* Joanna Fortuna
+* Michal Krawczyk
+* Mateusz Łaszczyk
```

Comment:
If you haven`t used polish letters in other names before, why did you used them in `Łaszczyk` name?

Reply:
My fault